lib/db/insert.go

```diff
@@ -0,0 +1,89 @@
+package db
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/shipherman/gophermart/ent"
+	"github.com/shipherman/gophermart/lib/acc"
+	"github.com/shipherman/gophermart/lib/models"
+)
+
+func InsertUser(newUser ent.User) error {
+	client := GetClient()
+	user, err := client.User.Create().
+		SetLogin(newUser.Login).
+		SetPassword(newUser.Password).
+		SetBalance(0).
+		SetWithdraw(0).
+		Save(context.Background())
+
+	fmt.Println(user, err)
+	if err != nil {
+		return err
+	}
+	return nil
```

Comment:
Как больше нравится, но можео сразу верунуть err
```suggestion
	return err
```